### ISSUE TITLE
Move do not disturb icon to quick settings area (see pictures)

### ISSUE THREAD

**Issue by CorvetteCole** [qwreey75/quick-settings-tweaks#42]
Currently in Gnome 43, the do not disturb icon is seen next to the date and time as seen below:
![Screenshot from 2022-11-08 12-08-58](https://user-images.githubusercontent.com/19213965/200630421-d2100bf8-39fa-4528-9742-c6a2edf6dce6.png)

I think that if notifications are combined in to the quick settings menu, then the do not disturb icon should also be moved to this area (seen below)
![Screenshot from 2022-11-08 12-09-03](https://user-images.githubusercontent.com/19213965/200630549-6ee7b764-ccf9-4e16-8d57-fcedca10397f.png)

Maybe next to the wifi icon on the far left?

**Comment by brunnogama** [comment]
Agreed!

**Comment by martindmtrv** [comment]
Agreed! The notification dot would go nicely in the right area, or maybe even if possible replace the dot with a small counter for notifications (though might be out of scope for this project)

![image](https://user-images.githubusercontent.com/36076588/202311025-29e0c3ef-73dc-4063-8203-33b3cf804ad7.png)

**Comment by sojusnik** [comment]
I think the best way would be to have an option where the "dot" and the "bell" icon should appear, regardless whether the "DND quick toggle" is enabled or not, as suggested [here](https://github.com/qwreey75/quick-settings-tweaks/issues/86).
